controllers/video.go

```diff
@@ -10,8 +10,8 @@ import (
 type variantsResponse struct {
 	Name               string `json:"name"`
 	Index              int    `json:"index"`
-	VideoBitrate       int    `json:"VideoBitrae"`
-	IsVideoPassthrough bool   `json:"IsVideoPassthrough"`
+	VideoBitrate       int    `json:"-"`
+	IsVideoPassthrough bool   `json:"-"`
```

Comment:
Your commit says you're changing them to not be exported, but they're still being exported.